framework/lib/components/table/index.ts

```diff
@@ -9,3 +8,4 @@ export {
   TableCaption,
   TableContainer,
 } from '@chakra-ui/react'
+export * from './table'
```

Comment:
`TableProps` should be exported as well.
I'll cover it on my workshop soon, but the main goal is to always export props, so that they can be used without creating "magic" types (like `ComponentProps<typeof Table>`, that's just ugly.)